content/data-structures/PS2D.h

```diff
@@ -11,19 +11,19 @@
 #pragma once
 
 struct PS2D {
-	vector<vector<long long>> ps;
+	vector<vector<ll>> ps;
 	int n, m;
-	PS2D(vector<vector<long long>>& a) {
-		n = size(a), m = size(a[0]);
-		ps = vector(n+1, vector<long long>(m+1));
+	PS2D(vector<vector<ll>>& a) {
+		n = sz(a), m = sz(a[0]);
+		ps = vector(n+1, vector<ll>(m+1));
 		for (int i = 0; i < n; i++) {
 			for (int j = 0; j < m; j++) {
 				ps[i+1][j+1] = a[i][j] + ps[i][j+1]
 				 + ps[i+1][j] - ps[i][j];
```

Comment:
is this a space before the plus sign or a tab character? doesnt look like a tab

Reply:
it is a space but I can remove it if necessary

Reply:
removed